finite_state_sdk/__init__.py

```diff
@@ -2011,10 +2011,11 @@ def update_finding_statuses(token, organization_context, user_id=None, finding_i
     return send_graphql_query(token, organization_context, mutation, variables)
 
 
-def upload_file_for_binary_analysis(token, organization_context, test_id=None, file_path=None,
-                                    chunk_size=1024 * 1024 * 1024 * 5, quick_scan=False):
+def upload_file_for_binary_analysis(
+    token, organization_context, test_id=None, file_path=None, chunk_size=1024**2 * 1000, quick_scan=False
```

Comment:
To enhance readability and maintainability, I recommend defining this value `chunk_size` as a constant at the beginning of the script or move to another module.